src/main/java/core/basesyntax/infrastructure/db/FileWriterImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.infrastructure.db;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class FileWriterImpl implements FileWriter {
+    private static final String FILE_NAME = "src/main/java/database.csv";
+
+    @Override
+    public void write(List<String> resultingReport) throws IOException {
```

Comment:
```suggestion
    public void write(String report) throws IOException {
```
pass completed report